src/substrait/textplan/converter/tests/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: Apache-2.0
+
+add_test_case(
+        textplan_conversion_test
+        SOURCES
+        BinaryToTextPlanConversionTest.cpp
+        EXTRA_LINK_LIBS
+        substrait_textplan_converter
+        substrait_common
+        parse_result_matchers
+        gmock
+        gtest
+        gtest_main)
+
+cmake_path(GET CMAKE_CURRENT_SOURCE_DIR PARENT_PATH TEXTPLAN_SOURCE_DIR)
+
+add_custom_command(
+  TARGET textplan_conversion_test
+  POST_BUILD
+  COMMAND ${CMAKE_COMMAND} -E echo "Copying unit test data.."
+  COMMAND ${CMAKE_COMMAND} -E make_directory "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/tests/data"
+  COMMAND ${CMAKE_COMMAND} -E copy
```

Comment:
Is this going to be cross platform?  E.g. will this work on Windows?

Reply:
The CMAKE side obviously will be Windows compatible.  Forward slashes are supposed to be okay for Windows.  If this ends up not working we can address it as a bug fix.

Reply:
I was wondering more about `echo`, `make_directory`, and `copy`.  Are those "cmake commands"?   Or is that cmake running something in bash?  (I suppose it would be mkdir in bash so this must be a cmake thing?)

Reply:
This equivalent to running cmake -E on the command line:
```
% cmake -E echo ttt  
ttt
% cmake -E copy a b
Error copying file "a" to "b".
```